data/flashcards.json

```diff
@@ -0,0 +1,102 @@
+[
+  {
+    "id": "1",
+    "englishWord": "apple",
+    "japaneseWord": "りんご",
+    "interval": 1,
+    "repetition": 0,
+    "easiness": 2.5,
+    "nextReview": "2024-09-26T00:00:00.000Z",
+    "audioSource": "../assets/audio/apple.mp3"
+  },
+  {
+    "id": "2",
+    "englishWord": "dog",
+    "japaneseWord": "犬",
+    "interval": 1,
+    "repetition": 0,
+    "easiness": 2.5,
+    "nextReview": "2024-09-26T00:00:00.000Z",
+    "audioSource": "../assets/audio/dog.mp3"
+  },
+  {
+    "id": "3",
+    "englishWord": "book",
+    "japaneseWord": "本",
+    "interval": 1,
+    "repetition": 0,
+    "easiness": 2.5,
+    "nextReview": "2024-09-26T00:00:00.000Z",
+    "audioSource": "../assets/audio/book.mp3"
+  },
+  {
+    "id": "4",
+    "englishWord": "cat",
+    "japaneseWord": "猫",
+    "interval": 1,
+    "repetition": 0,
+    "easiness": 2.5,
+    "nextReview": "2024-09-26T00:00:00.000Z",
+    "audioSource": "../assets/audio/cat.mp3"
```

Comment:
I don't think we can load sound like this because of the way production assets are bundled they will no longer be on this relative path. I noticed the sound wasn't played in the production preview for this reason. 

I think let's just not do sound for the time being. We'll also probably want to generate sound than play from files on our server.

Reply:
I see, yes last night I had a quick look at the the production preview and noticed the same. I'll remove the sound logic for now and maybe just leave the button so we can visualize the layout.

BTW the audio I used was generated by [https://elevenlabs.io/](url)